browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -318,20 +259,8 @@ void OnBeforeURLRequestAdBlockTP(const ResponseCallback& next_callback,
   scoped_refptr<base::SequencedTaskRunner> task_runner =
       g_brave_browser_process->ad_block_service()->GetTaskRunner();
 
-  SecureDnsConfig secure_dns_config =
-      SystemNetworkContextManager::GetStubResolverConfigReader()
-          ->GetSecureDnsConfiguration(false);
-
-  bool doh_enabled = (secure_dns_config.mode() == net::SecureDnsMode::kSecure);
-
-  // DoH or standard DNS queries won't be routed through Tor, so we need to
-  // skip it.
-  // Also, skip CNAME uncloaking if there is currently a configured proxy.
-  bool should_check_uncloaked =
-      base::FeatureList::IsEnabled(
-          brave_shields::features::kBraveAdblockCnameUncloaking) &&
-      ctx->browser_context && !ctx->browser_context->IsTor() &&
```

Comment:
Actually we can't just remove it like this because DNS requests can't be routed through Tor connection.
1. Tor can't be used to route UDP traffic.
2. If we forced DoH, it will create [proxy deadlock](https://github.com/brave/brave-core/pull/7769/commits/5e099e88e02907d4553adf4eb04590113f703233#diff-3ce0a287b7804c3416893f38bed22d098d3c8756b659ca5915ca088bb674434eR8) and DoH is also leaking from Tor perspective.